custom_components/opnsense/config_flow.py

```
@@ -268,131 +288,305 @@
+                    CONF_GRANULAR_SYNC_OPTIONS,
+                    fallback.get(CONF_GRANULAR_SYNC_OPTIONS, DEFAULT_GRANULAR_SYNC_OPTIONS),
+                ),
+            ): selector.BooleanSelector(selector.BooleanSelectorConfig()),
+        }
+    )
+
+
+def _build_granular_sync_schema(
+    user_input: MutableMapping[str, Any] | None,
+    fallback: MutableMapping[str, Any] | None = None,
+    reconf: bool = False,
```

Pylint (3.3.7)

[warning] Unused argument 'reconf'

(W0613)